openshift/3scale_backend.conf

```
@@ -5,6 +5,11 @@ def parse_int_env(env_name)
   !env_var.nil? && !env_var.empty? ? env_var.to_i : nil
 end
 
+def parse_boolean_env(env_name)
+  value = ENV[env_name]
+  !value.nil? && value.downcase == 'true'
```

Why not also check for the string `"1"`?
```suggestion
  value == '1'.freeze || value&.downcase == 'true'.freeze
```

---

OK. Committed your suggestion :+1: